data/config_hypr.yaml

```diff
@@ -0,0 +1,3 @@
+HYPR_TOKEN: "<hyper API Token>"
```

Comment:
config file vs env variable.. was there any specific preference to use config file instead of env ? With this config file approach looks like all the docker compose files needs to be updated with this file mount.. My preference would be to have a section at the end in env file itself.. If there are other thoughts/opinions, let's discuss before making this change.

Reply:
Good question! I started to put in the env vars file. We can still do that. @dmartin5470 felt that the env vars were getting heavy. So as we add providers we can add a config file for them and prevent env var file from getting too large. By keeping everything in env vars it does simplify the implementation and readme instructions a little.

Reply:
My thought was to keep these configurations a bit more descriptive and separate if we can. Especially if we want to reintroduce the okta integration (and potentially add others in the future). I feel that env variables are fine when there is a one-to-one correlation (e.g. enable this by setting this env variable). But these mfa integrations have potential to be more complicated with many variables needed just for them to work. 

But @ce-rbabu if you think from an integration perspective (since you've been interacting with more people who are actually trying to set these up in their own use cases) that env variables are easier, we can roll with that.

Reply:
If there are just 3 configuration options, I'd go with env variables

Reply:
@renjith85 Discussed on daily. I will use env vars.